rollout-renku-version/rollout-renku-deployment.sh

```diff
@@ -27,15 +30,14 @@ git config --global user.name "$GIT_USER"
 # update renku version and push
 git checkout -b auto-update/${CHART_NAME}-${CHART_VERSION} ${UPSTREAM_BRANCH}
 
-PRODUCTION_DIR="gitops/production"
 
 # use current date
 date=$(date +"%B %dth %Y")
 clusters=$(ls -d ${PRODUCTION_DIR}/*)
 
 for cluster in $clusters
 do
-  if [[ ! $cluster =~ "rancher" && ! $cluster =~ "renku-dev" ]];
+  if [[ ! $cluster =~ $RANCHER_DIR && ! $cluster =~ $DEV_DIR ]];
```

Comment:
I think we should assume that other clusters could be exempted from automatic updates so I would make it a list and not specific to these two.